pipelines/curate_bridge.py

```
@@ -183,327 +176,257 @@ def _evidence_hash(value: Any) -> str:
-def _explicit_order_fields(record: dict[str, Any], events: list[Any]) -> list[str]:
-    found: set[str] = set()
-    containers: list[dict[str, Any]] = [record]
-    meta = record.get("meta")
-    if isinstance(meta, dict):
-        containers.append(meta)
-    containers.extend(event for event in events if isinstance(event, dict))
-    for container in containers:
-        found.update(key for key in EXPLICIT_ORDER_KEYS if key in container)
-    return sorted(found)
 
+def raster_sidecar(record: Any) -> tuple[str | None, Any]:
```

Ruff (0.16.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `record`

(ANN401)